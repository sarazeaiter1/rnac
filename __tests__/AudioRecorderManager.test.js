// import React from 'react';
// import { Text, View, Button } from 'react-native';
// import renderer from 'react-test-renderer';
// import { shallow } from 'enzyme';
import AudioRecorderManager from '../src/js/native-interfaces/AudioRecorderManager';
describe('AudioRecorderManager', () => {
  test('exports', () => {
    expect(AudioRecorderManager).toMatchSnapshot();
  });
});
