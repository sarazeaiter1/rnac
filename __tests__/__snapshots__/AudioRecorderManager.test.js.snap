// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AudioRecorderManager exports 1`] = `
Object {
  "prepareRecordingAtPath": [Function],
  "startRecording": [Function],
  "stopRecording": [Function],
}
`;
